### PR TITLE
Release/1.7.0

### PR DESCRIPTION
**Pull request by zacoid55**
#### What's in this PR?

This PR contains all the latest changes that are going into 1.7.0

---

#### Pre-merge checklist

Before merging any PR, please check the following common things that should be done beforehand. These aren't all always required, so just check the box if it doesn't apply.

- [x] **When adding files, make sure they're added to the right target**. If you're adding new files that should be bundled up with Cocoapods etc, they need to be added to the `TABTestKit` target, not `Pods-TABTestKit_Example` etc.
- [x] **Run `pod install` to ensure that the latest changes are in the Example project**. Without this, Carthage might not see the latest changes.
- [x] **Added and updated tests where possible**. This isn't always possible but try wherever you can. The example app contains UI tests to test many of the TABTestKit features.
- [x] **Updated the `CHANGELOG`**. For any changes pending a release, add to the Pending section. For releases, move everything pending to the release section.
- [x] **Updated the `README`**. Add info for any new features, update existing info for anything that's changed or needs extra info.

### CODE REVIEW

File: Example/Pods/Pods.xcodeproj/project.pbxproj.orig [hunk 1]
```diff
@@ -0,0 +1,1509 @@
+// !$*UTF8*$!
```
Comment (roger-tan) [inline Example/Pods/Pods.xcodeproj/project.pbxproj.orig:1]: We need to delete this file.

Comment (zacoid55) [reply Example/Pods/Pods.xcodeproj/project.pbxproj.orig]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # CHANGELOG
 
-## Pending
+## 1.7.0
+
+- Added support for double tapping, long pressing and two finger tapping, with default implementations for any Elements that conform to Tappable. (issue #88)
+- Added Swift Package Manager support
+- Added a `Refreshable` protocol so for each element/screen that conforms to protocol `refresh` function can be defined 
+- Added a `DatePicker` element. (issue #55)
```
Comment (SamDKBerry) [inline CHANGELOG.md:9]: `value` can now be accessed directly from elements

Comment (zacoid55) [reply CHANGELOG.md:10]: Is this okay, @SamDKBerry ?

Comment (SamDKBerry) [reply CHANGELOG.md:10]: Looks good

**Pull request merged**